### PR TITLE
adding doc to distortPoints

### PR DESCRIPTION
**Pull request by ericmariasis**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [x] There is a reference to the original bug report and related work
- [x] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by ericmariasis** [comment]
Resolves [#25352](https://github.com/opencv/opencv/issues/25352).

File: modules/calib3d/src/fisheye.cpp [hunk 1]
```diff
@@ -253,7 +253,11 @@ void cv::fisheye::projectPoints(InputArray objectPoints, OutputArray imagePoints
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////
 /// cv::fisheye::distortPoints
-
+// Note:
+//   The input is normalized points and the output is raw points.
+//   The inputs must have the optical center subtracted
+//   and then divided by focal length.
+//   While the output is multiplied by focal length with optical center added.
```
Comment (asmorkalov) [inline modules/calib3d/src/fisheye.cpp:260]: It's equivalent to the documentation note: `Note that the function assumes the camera intrinsic matrix of the undistorted points to be identity. This means if you want to distort image points you have to multiply them with or use another function overload.` See https://docs.opencv.org/4.x/db/d58/group__calib3d__fisheye.html#ga75d8877a98e38d0b29b6892c5f8d7765
Also I just added overload to handle undistort camera matrix: https://github.com/opencv/opencv/pull/25943. I do not think that the note is useful.

Comment (ericmariasis) [reply modules/calib3d/src/fisheye.cpp:260]: Wait so are you saying this change I made isn't needed and we should close the reported issue?

Comment (asmorkalov) [reply modules/calib3d/src/fisheye.cpp:260]: yes

**Pull request closed without merging**